packages/next/src/server/lib/start-server.ts

```diff
@@ -258,13 +259,24 @@ export async function startServer(
           process.exit(code ?? 0)
         }
         const exception = (err: Error) => {
+          if (isPostpone(err)) {
+            // React postpones that are unhandled might end up logged here but they're
+            // not really errors. They're just part of rendering.
+            return
+          }
+
           // This is the render worker, we keep the process alive
           console.error(err)
         }
         process.on('exit', (code) => cleanup(code))
         // callback value is signal string, exit with 0
         process.on('SIGINT', () => cleanup(0))
         process.on('SIGTERM', () => cleanup(0))
+        process.on('rejectionHandled', () => {
```

Comment:
@feedthejim We'd also need this in the worker because otherwise it logs this as an error to the console.